### PR TITLE
feat(docker): Add CloudOps Dockerfile & CircleCI build instructions

### PR DESCRIPTION
**Pull request by jbuck**
Do not merge yet, blocked on #4191

### CODE REVIEW

**Comment by vladikoff** [comment]
Can we clean up the https://github.com/mozilla/fxa-content-server/blob/master/Dockerfile file. 

No one is using that one.

**Comment by jbuck** [comment]
Yep, I can replace `Dockerfile` with `Dockerfile-build`

**Comment by vladikoff** [comment]
> Yep, I can replace Dockerfile with Dockerfile-build

We might wanna call them `Dockerfile-build` to be consistent with other repos, but up to you.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8f111e4d472b4adbeb40a352854ef34f48423b2a
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by vladikoff** [comment]
Needs rebase.

**Comment by rfk** [comment]
bump @jbuck, as the linked blocker #4191 appears to have been resolved

File: Dockerfile-build [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM node:4.5.0
```
Comment (vladikoff) [inline Dockerfile-build]: @jbuck needs update to 4.6  here?

**Comment by vladikoff** [comment]
Please reopen this once we find the time to add this 👍

**Pull request closed without merging**